desktop/src/app/styles/vendor/material-theme.scss

```diff
@@ -197,6 +197,10 @@ mat-button-toggle-group {
     }
 }
 
+mat-calendar .mat-calendar-table-header {
+    color: rgba(0, 0, 0, 0.87);
```

Comment:
Is there an existing theme color that can be used for this instead? Or maybe we should add this color to our themes?

Reply:
That's a very good question - I'll check.

Reply:
Looks like the calendar control isn't themed. It shows up black and white no matter what the theme is.